bin/schematic2layout.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+import argparse
+import align
+
+if __name__ == '__main__':
```

Comment:
Recently I've found it better to have the command line parser as part of the "package" so we can run command line level tests.
Then if file just becomes, generate the parser (one function in align somewhere) and the call.

Reply:
Good point. See align/cmdline.py

Reply:
bin/schematic2layout.py now simply reads as:

`if __name__ == '__main__':
    align.CmdlineParser().parse_args()`